.github/workflows/ci.yml

```diff
@@ -31,3 +31,18 @@ jobs:
     - name: Run other tests
       run: |
         dotnet test
+
+  build_with_fsharp_from_mono:
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build
+      run: |
+        DEBIAN_FRONTEND=noninteractive sudo apt install -y msbuild fsharp nuget
```

Comment:
Would you specify the version for these packages? I think you can pass it to apt command by `apt install -y msbuild=<version>`

Reply:
why would you want to do this?

Reply:
Because the purpose of this job is to demonstrate that it can be compiled on a specific fsharp version? The "latest" version  might be changed by runtime and it may be difficult to reproduce when the build is broken.
(TBH I don't have strong opinion about this though. I can merge as is if you feel not to specify.)

Reply:
Ah I suspected that you might be thinking along those lines. However, the point of this compatibility is that DNL.Core stays compatible to the last version of fsharp's apt package provided by Mono's apt sources. If they upgrade fsharp package to a higher version, then that's not a bad thing; I mean, then this CI would not be checking anymore against v4.5 but against the last version they bundle, but in case it lags behind (again) compared to the last version of F# provided by .NETCore, then the CI becomes useful again.

On top of that, specifying a version of the package might even not work at all. Don't quote me on this but I think that, whenever an apt source is upgraded with a new version of a package, the old version doesn't exist anymore (and cannot be retrieved). Maybe the only useful thing that comes of specifying a version when doing an apt installation is in case there are two different apt sources enabled, which provide the same package but with different version.

(That being said, there is actually an fsharp package with version older than 4.5 -- IIRC it's 4.0 -- which comes from Canonical repos, as opposed to Mono's repos. However, IME this version is not compatible with msbuild.)

Reply:
I see.